cli/src/description_util.rs

```diff
@@ -28,8 +29,17 @@ where
 {
     let description = lines
         .into_iter()
-        .filter(|line| !line.as_ref().starts_with("JJ:"))
-        .fold(String::new(), |acc, line| acc + line.as_ref() + "\n");
+        .fold_while(String::new(), |acc, line| {
+            let line = line.as_ref();
+            if line == "JJ: ignore-rest" {
```

Comment:
nit: maybe better to trim trailing whitespace and `-` as we do for `JJ: describe "` line?

Reply:
The detection of the `"JJ: describe "` is just `line.strip_prefix(...)`, so I changed it to that.

Reply:
It's a bit stricter in that "JJ: ignore-rest<SPACE>" is required (and would fail to parse if trailing argument contained junk.)

BTW, `.strip_prefix()` can be replaced with `.starts_with()`.